README.md

```diff
@@ -1,22 +1,19 @@
 homebrew-font
 =============
 
-# Install fonts
+# Installing Ricty fonts
 
-Compile Ricty Font
-
-* Install
-   * Recommend
+Recommended installation by tapping the repository
 
 ```
-$ brew tap sanemat/font
+$ brew tap sanemat/homebrew-font
```

Comment:
oh, `brew tap sanemat/homebrew-font` and `brew tap sanemat/font`, both are good.
https://github.com/Homebrew/homebrew/blob/3cebc21bb6a493cd96a679173f8c4b1ce8827125/share/doc/homebrew/brew-tap.md#naming-conventions-and-limitations